rpcs3/Emu/system_progress.cpp

```diff
@@ -36,18 +37,188 @@ namespace rsx::overlays
 	};
 } // namespace rsx::overlays
 
+scoped_progress_dialog::scoped_progress_dialog(std::string text) noexcept
+{
```

Comment:
Please move this to another file. It's way too large

Reply:
Done